### PR TITLE
use standard license on new files

### PR DESCRIPTION
**Pull request by pjfanning**
relates to https://github.com/apache/incubator-pekko/pull/474

### CODE REVIEW

**Review by nvollmar — APPROVED** [review]
lgtm

File: project/AddMetaInfLicenseFiles.scala [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
  * Licensed to the Apache Software Foundation (ASF) under one or more
- * license agreements; and to You under the Apache License, version 2.0:
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
  *
- *   https://www.apache.org/licenses/LICENSE-2.0
+ *    http://www.apache.org/licenses/LICENSE-2.0
```
Comment (jrudolph) [inline project/AddMetaInfLicenseFiles.scala:9]: Was that intended?

Comment (pjfanning) [reply project/AddMetaInfLicenseFiles.scala:9]: * the standard ASL header uses http
* see https://www.apache.org/legal/src-headers.html

Comment (mdedetrich) [reply project/AddMetaInfLicenseFiles.scala:9]: Also note that it redirects to https, but yes we should be using the exact same text that is references on the official ASF website (https or not).

Comment (jrudolph) [reply project/AddMetaInfLicenseFiles.scala:9]: :+1: thanks for clarifying.

**Pull request merged**